inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/ActiveLearningServiceImpl.java

```diff
@@ -44,6 +46,12 @@
 {
     private final DocumentService documentService;
     private final RecommendationService recommendationService;
+    private List<AnnotationObject> recommendations;
+    private List<List<AnnotationObject>> listOfRecommendationsForEachToken;
+    private AnnotatorState annotatorState;
+    private AnnotationLayer selectedLayer;
```

Comment:
This state must not be in the service since the service is shared between all users. It must be in the user's state.
```
     private List<AnnotationObject> recommendations;
     private List<List<AnnotationObject>> listOfRecommendationsForEachToken;
     private AnnotatorState annotatorState;
     private AnnotationLayer selectedLayer;
```

Reply:
These fields are not used. I forgot to remove them. Now, these fields are not there.